src/services/fileStorage/proxy-service.js

```diff
@@ -644,6 +644,7 @@ const newFileService = {
 		}, params)
 			.then((signedUrl) => {
 				const headers = signedUrl.header;
+				if (signedUrl instanceof Error) throw new GeneralError('No file service available.');
```

Comment:
der signedUrlService sollte eigentlich bei einem Fehler nicht hier landen...?!?

Reply:
Genau das ist das Problem. Zumindest bei mir (node 11.15.0) tut es genau das. Die Fehler, die in aufgerufenen Funktionen geworfen werden, werden nicht an den Client weitergegeben. Stattdessen landet der Error im Result-Objekt `signedUrl`.

Reply:
okay, wir nutzen noch node 8.15 - bitte nutze nvm und setze dann via nvm use im verzeichnis die in der .nvmrc definierte node version.

Reply:
Ich glaube das Problem sollte an der Wurzel angegangen werden: `SignedUrlService#create` returnt Fehler,  anstatt sie zu werfen (`throw`)